qcodes_contrib_drivers/drivers/Keysight/Keysight_J7211.py

```diff
@@ -0,0 +1,38 @@
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.utils.validators import Ints
+from typing import Optional
+
+
+class Keysight_J7211(VisaInstrument):
+    r"""
+    Qcodes driver for the Keysight J7211 Attenuation Control Unit.
+    Tested with J7211B.
+
+    Args:
+        name: Instrument name
+        address: Address or VISA alias of instrument
+        attenuation: Optional attenuation level (in dB) to set on startup
+    """
+
+    def __init__(self, name: str, address: str,
+                 attenuation: Optional[int] = None, **kwargs):
+        super().__init__(name=name, address=address, terminator='\r', **kwargs)
```

Comment:
please put the terminator to the init of the driver such that it's possible to change it if needed.
```suggestion
    def __init__(self, name: str, address: str,
                 attenuation: Optional[int] = None, terminator='\r', **kwargs):
        super().__init__(name=name, address=address, terminator=terminator, **kwargs)
```